README.md

```diff
@@ -1148,6 +1148,20 @@ $ make run-smoke-tests [key=value]*
 It will run only tests tagged with `@smoke`.
 All options from BDD tests do also apply here.
 
+#### Running performance tests
```

Comment:
Please add the chapter to the index in the beginning of the readme.

Reply:
Done.